components/TextFields/tests/unit/TextFieldControllerClassPropertiesLegacyTests.swift

```diff
@@ -356,4 +363,83 @@ class TextFieldControllerClassPropertiesLegacyTests: XCTestCase {
     XCTAssertEqual(controller.trailingUnderlineLabelFont,
                    MDCTextInputControllerLegacyFullWidth.trailingUnderlineLabelFontDefault)
   }
+  
+  func testLegacyDefaultTextInputClearButtonTintColorUsesDefault() {
+    // Given
+    MDCTextInputControllerLegacyDefault.textInputClearButtonTintColorDefault = .orange
+    
+    // When
+    let textInput = MDCTextField()
+    let controllerLegacyDefault = MDCTextInputControllerLegacyDefault(textInput: textInput)
+    
+    // Then
+    XCTAssertEqual(controllerLegacyDefault.textInputClearButtonTintColor,
+                   MDCTextInputControllerLegacyDefault.textInputClearButtonTintColorDefault)
+  }
+  
+  func testLegacyDefaultTextInputClearButtonTintColorDefaultAppliesToTextField() {
+    // Given
+    MDCTextInputControllerLegacyDefault.textInputClearButtonTintColorDefault = .orange
+    
+    // When
+    let textInputLegacyDefault = MDCTextField()
+    let _ = MDCTextInputControllerLegacyDefault(textInput: textInputLegacyDefault)
+    
+    // Then
+    XCTAssertEqual(textInputLegacyDefault.clearButton.tintColor,
+                   MDCTextInputControllerLegacyDefault.textInputClearButtonTintColorDefault)
+  }
+  
+  func testLegacyDefaultTextInputClearButtonTintColorAppliesToTextField() {
+    // Given
+    let textInputLegacyDefault = MDCTextField()
+    let controllerLegacyDefault = MDCTextInputControllerLegacyDefault(textInput: textInputLegacyDefault)
```

Comment:
I would love to see all these not with `MDCTextInputControllerLegacy...anything` and instead `MDCTextInputControllerFilled` and `MDCTextInputControllerOutline`.

Reply:
The test cases like these for `MDCTextInputControllerFilled` and `MDCTextInputControllerOutline` are in #4449 (unless you were thinking of different test cases?). Should I remove the legacy tests altogether?

Reply:
Spoke offline. Learned that part is already done!